### PR TITLE
Delete methadone, use rake task instead

### PR DESCRIPTION
**Pull request by AntonKolmakov**
#36

### CODE REVIEW

**Comment by tuwilof** [comment]
need delete
https://github.com/funbox/tomograph/blob/master/spec/exe/tomograph_spec.rb
otherwise tests fail

```
$ bundle exec rake
273/home/travis/.rvm/rubies/ruby-2.4.10/bin/ruby -I/home/travis/.rvm/gems/ruby-2.4.10/gems/rspec-core-3.9.3/lib:/home/travis/.rvm/gems/ruby-2.4.10/gems/rspec-support-3.9.3/lib /home/travis/.rvm/gems/ruby-2.4.10/gems/rspec-core-3.9.3/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb
274bundler: command not found: ./exe/tomograph
275Install missing gem executables with `bundle install`
276Fbundler: command not found: ./exe/tomograph
277Install missing gem executables with `bundle install`
278Fbundler: command not found: ./exe/tomograph
279Install missing gem executables with `bundle install`
280F.............invalid option: --pattern
281
282
283Failures:
284
285  1) tomograph in default mode produces correct json
286     Failure/Error: expect(run_tomograph).to be_truthy
287     
288       expected: truthy value
289            got: false
290     # ./spec/exe/tomograph_spec.rb:16:in `block in run_tests_with_file'
291     # ./spec/exe/tomograph_spec.rb:15:in `run_tests_with_file'
292     # ./spec/exe/tomograph_spec.rb:25:in `block (3 levels) in <top (required)>'
293
294  2) tomograph with --exclude-description produces correct json
295     Failure/Error: expect(run_tomograph).to be_truthy
296     
297       expected: truthy value
298            got: false
299     # ./spec/exe/tomograph_spec.rb:16:in `block in run_tests_with_file'
300     # ./spec/exe/tomograph_spec.rb:15:in `run_tests_with_file'
301     # ./spec/exe/tomograph_spec.rb:34:in `block (3 levels) in <top (required)>'
302
303  3) tomograph with --split produces correct json
304     Failure/Error: expect(run_tomograph).to be_truthy
305     
306       expected: truthy value
307            got: false
308     # ./spec/exe/tomograph_spec.rb:50:in `block (3 levels) in <top (required)>'
309
310Finished in 0.56794 seconds (files took 0.17134 seconds to load)
31116 examples, 3 failures
312
313Failed examples:
314
315rspec ./spec/exe/tomograph_spec.rb:24 # tomograph in default mode produces correct json
316rspec ./spec/exe/tomograph_spec.rb:33 # tomograph with --exclude-description produces correct json
317rspec ./spec/exe/tomograph_spec.rb:49 # tomograph with --split produces correct json
318
319Coverage report generated for RSpec to /home/travis/build/funbox/tomograph/coverage. 757 / 904 LOC (83.74%) covered.
320SimpleCov failed with exit 1
321/home/travis/.rvm/rubies/ruby-2.4.10/bin/ruby -I/home/travis/.rvm/gems/ruby-2.4.10/gems/rspec-core-3.9.3/lib:/home/travis/.rvm/gems/ruby-2.4.10/gems/rspec-support-3.9.3/lib /home/travis/.rvm/gems/ruby-2.4.10/gems/rspec-core-3.9.3/exe/rspec --pattern spec/\*\*\{,/\*/\*\*\}/\*_spec.rb failed
322The command "bundle exec rake" exited with 1.
323
324
325Done. Your build exited with 1.
```

**Comment by tuwilof** [comment]
hmm, build failed anyway

**Comment by AntonKolmakov** [comment]
> hmm, build failed anyway

Yeh, seems to something wrong with rake test.

**Review by tuwilof — APPROVED** [review]
🙆‍♂️

File: README.md [hunk 1]
```diff
@@ -39,20 +39,20 @@ tomogram = Tomograph::Tomogram.new(drafter_yaml_path: '/path/to/doc.yaml')
...
 ```bash
-tomograph -d 4 doc_by_drafter4.yaml doc.json
+rake tomograph -- --drafter=4 --input=doc_by_drafter4.yaml --output=doc.json
 ```
 
 Run CLI with `-h` to get detailed help:
 
 ```bash
-tomograph -h
+rake tomograph -- -h
```
[the remark this answers is not in the document]
Comment (AntonKolmakov) [reply README.md:55]: find here: https://stackoverflow.com/a/19466362

**Comment by tuwilof** [comment]
it turned out to use rake task from gem without rails such a pain 😮

**Pull request merged**